Configuration/PyReleaseValidation/test/BuildFile.xml

```diff
@@ -2,3 +2,29 @@
 </test>
 <test name="test-runTheMatrix-interactive" command="test-runTheMatrix_interactive.sh">
 </test>
+
+<!-- Tests for MC production for 2022 and 2023 -->
+<!-- 
+    The conditions used here for 2022 and 2023 will be changed 
+    to use the proper GT from ALCA including the BS smearing 
+    record from GT. For the moment we use the automatic ones.
+    By default these tests won't fail if a failure happens at
+    HLT step. If you want to catch those errors please set the
+    environment variable CMSSW_MC_SETUP_TEST_CATCH_HLT.
+-->
+
+<!-- In CMSSW_13_0_18 the auto:phase1_2023_realistic (pre BPix) is 130X_mcRun3_2023_realistic_postBPix_v1 -->
+<test name="test_MC_23_setup" command="test_mc_setup/test_MC_setup.sh 140X_mcRun3_2023_realistic_v3 Run3_2023 2023v12 CMSSW_13_0_18 130X_mcRun3_2023_realistic_v13 Realistic25ns13p6TeVEarly2023Collision" />
```

Comment:
I mean this GT, 140X_mcRun3_2023_realistic_v3, should it be phase1_2023_realistic instead? Same for 2022 Iphase1_2022_realistic).

Reply:
The GT for HLT release is more-or-less OK for hard-coding, I think. The same reason as the release.

Reply:
Maybe the 14_0_X backport should use the `140X_mcRun3_2023_realistic_v3` GT explicitly?

Reply:
Indeed I think the `14_0_X` would be better to use an hardcoded GT, but also yes for `master` we could use the automatic ones everywhere.

Reply:
Does it sound reasonable?

Reply:
I think using the symbolic GT is a good choice, so we see the error immediately when something was wrong. Note that, '140X_mcRun3_2023_realistic_v3', is also 'phase1_2023_realistic' in 14_X. So carbon-copy backport should be OK also,
https://github.com/cms-sw/cmssw/blob/CMSSW_14_0_X/Configuration/AlCa/python/autoCond.py

Reply:
So you mean keep the HLT running with a different older GT and have it possibly crash (presumably)?

Reply:
Symbolic GT is for GEN-SIM-DIGi and RECO step. Do I misunderstand something?

Reply:
Sure but the scheme would be:

1. GEN-SIM-DIGI with symbolic in `master` (now `141X`);
2. HLT with `130X` or `124X`
3. RECO with symbolic as 1.

Now, once the issues of forward compatibility are solved, wouldn't this setup risk to crash anyway because the GT in (2) is not granted to be compatible with the GT in (1)?

Reply:
What HLT is needed from HLT should not have a big change I assume. At least it keeps the same condition as that year. From Run-2 UL experience, the HLT GT is there for very long.

Reply:
> Now, once the issues of forward compatibility are solved, wouldn't this setup risk anyway to crash because the GT in (2) is not granted to be compatible with the GT in (1)?

Wouldn't such a crash be a good thing (by showing the GTs have become incompatible)? (assuming at some point in the future the GTs would have to be made compatible again)

Reply:
Are really GTs for specific years meant (and granted) to be compatible among releases? I have really no clue (@cms-sw/alca-l2?).

Reply:
Thanks Marco. This would settle this (from my point of view).

But, there's a subsequent issue. In `14_0_X` we can't run the GEN-SIM step (at least) with a `130X_mcRun3_2023_realistic_v*` (tested up to the last `v15`)  GT. Since a record needed for  PPS is not there (`CTPPSInterpolatedOpticsRcd`).

```
----- Begin Fatal Exception 24-Apr-2024 18:02:52 CEST-----------------------
An exception of category 'NoRecord' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'RAWSIMoutput_step'
   [2] Prefetching for module PoolOutputModule/'RAWSIMoutput'
   [3] Prefetching for module OscarMTProducer/'g4SimHits'
   [4] Prefetching for module PPSSimTrackProducer/'LHCTransport'
   [5] Calling method for EventSetup module CTPPSInterpolatedOpticalFunctionsESSource/'ctppsInterpolatedOpticalFunctionsESSource'
   [6] While getting dependent Record from Record CTPPSInterpolatedOpticsRcd
Exception Message:
No "LHCInfoPerLSRcd" record found in the EventSetup.

 Please add an ESSource or ESProducer that delivers such a record.
----- End Fatal Exception -------------------------------------------------
```

So in order to run this properly new uniform GTs (`140X` + `130X` GTs for 2023 and 140X` + `124X`   GTs for 2022, I haven't tested for 2022 but I suspect a similar issue) would be needed. Now to get things rolling, would anyway make sense to have this setup merged using the *wrong* GTs at least to have core experts working on the fix for the forward compatibility (that was the original objective here)? With the caveat and a reminder (==issue) that we need new GTs anyway? Or do you prefer to wait for them?

Reply:
Hi @AdrianoDee 
I admit I confuse. The only change I propose is to change the GT used in GEN-SIM-DIGI, and RECO step to symblic one. So, technically, it does not change anything. I don't propose to use 13_0 GT with 14_0, 14_1 release. Do I miss anything?